waspc/data/Generator/templates/sdk/server/api/index.ts

```diff
@@ -2,8 +2,6 @@
 
 import { type ParamsDictionary as ExpressParams, type Query as ExpressQuery } from 'express-serve-static-core'
 
-export type { ParamsDictionary as ExpressParams, Query as ExpressQuery } from 'express-serve-static-core'
```

Comment:
I checked, nobody is importing this anywhere in our codebase + they are not part of our public docs + they can be imported from `express-serve-static-core` if needed, so I removed these exports.